src/device/usbd.c

```diff
@@ -1232,6 +1232,8 @@ bool usbd_edpt_xfer(uint8_t rhport, uint8_t ep_addr, uint8_t * buffer, uint16_t
   // TU_VERIFY(tud_ready());
 
   TU_LOG_USBD("  Queue EP %02X with %u bytes ...\r\n", ep_addr, total_bytes);
+  TU_LOG3_BUF(buffer, total_bytes);
```

Comment:
must check if this is an IN transfer, otherwise, data contents is meaningless. It must be wrapped in `#if CFG_TUD_LOG_LEVEL >= 3`